src/index.html

```diff
@@ -1,13 +1,444 @@
+            </div>
+
+            <div class="grid__item grid__item--7-12">
+              <a href="#testimonials" class="about__subtitle link">
+                Over 50 years experience in love.
+              </a>
+
+              <p class="about__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+              </p>
+
+              <p class="about__paragraph">
+                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.
+              </p>
+
+              <p class="about__paragraph about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="gallery">
+        <div class="container">
+          <h2 class="page__section-title">
+            Gallery
+          </h2>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-8">
+                  <div class="place__photo-container">
+                    <img
+                    src="./images/photos/1.jpg"
+                    alt="Place 1"
+                    class="place__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-9-12">
+                  <div class="place__photo-container">
+                    <img
+                    src="./images/photos/2.jpg"
+                    alt="Place 1"
+                    class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="place__description">
+              <a
+                href="https://goo.gl/maps/PEbVCgDY5ZwUUNB69"
+                class="place__link link"
+                target="_blank"
+              >
+                Marina Palms
+              </a>
+              / North Miami Beach, FL 33162
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-6">
+                  <div class="place__photo-container">
+                    <img
+                    src="./images/photos/3.jpg"
+                    alt="Place 2"
+                    class="place__photo"
+                    >
```

Comment:
Pay attention to code style

```suggestion
                    <img
                      src="./images/photos/3.jpg"
                      alt="Place 2"
                      class="place__photo"
                    >
```